src/main/java/Duke.java

```diff
@@ -1,10 +1,186 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String inputStatement;
+        int endFlag = 0;
+        String separatingLine = "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~";
+        Task[] listOfItems = new Task[100];
+        int listWordCount = 0;
+
+        printWelcomeText(separatingLine);
+        printBetweenLines(separatingLine, "Greetings, care for a cup of coffee?");
+
+        //loop begins:
+        while (endFlag == 0) {
+            Scanner in = new Scanner(System.in);
+            inputStatement = in.nextLine();
+            if (inputStatement.equalsIgnoreCase("bye")) {
+                //signals end of programme
+                endFlag = 1;
+            }
+            else if (inputStatement.equalsIgnoreCase("list") ){
+                System.out.println(separatingLine);
+                //makes sure list is not empty
+                if (checkListForItems(separatingLine, listOfItems[0] == null)) continue;
+                //print out list of items
+                System.out.println("Here's what you have:");
+                printArrangedList(listOfItems);
+                System.out.println(separatingLine);
+            }
+            else if(inputStatement.startsWith("done")){
+                System.out.println(separatingLine);
+                markAsDone(inputStatement, separatingLine, listOfItems);
+            }
+            else {
+                //proceeds to add task if task is valid
+                listWordCount = classifyAndAddTask(inputStatement, separatingLine, listOfItems, listWordCount);
+            }
+        }
+        //loop ends
+
```

Comment:
Too Much space is added.
Suggestion: Remove unnecessary spacing to improve readability of codes